scripts/gen-ceph-kustomize.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+# set -ex
+
+# expect that the common.sh is in the same dir as the calling script
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+. ${SCRIPTPATH}/common.sh --source-only
+
+if [ -z "$IMAGE" ]; then
+    echo "Unable to determine ceph image."
+    exit 1
+fi
+
+if [ -z "$NAMESPACE" ]; then
+    echo "Please set NAMESPACE"; exit 1
+fi
+
+if [ ! -d ${DEPLOY_DIR} ]; then
+    mkdir -p ${DEPLOY_DIR}
+fi
+
+pushd ${DEPLOY_DIR}
+
+HOSTNETWORK=${HOSTNETWORK:-false}
+
+POOLS=("volumes" "images")
+
+function add_ceph_pod {
+cat <<EOF >ceph-pod.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ceph
+  namespace: openstack
+  labels:
+    app.kubernetes.io/name: ceph
+    app: ceph
+spec:
+  hostNetwork: $HOSTNETWORK
+  containers:
+   - image: quay.io/ceph/daemon:latest-quincy
+     name: ceph
+     env:
```

Comment:
@Akrog I didn't find anything to push/force the FSID, and we use that value in the Cinder CR. I'm trying to think about a potential development flow, but I suspect we need something to build the cinder config snippet and inject it to the CR

Reply:
long term I guess we can try adding the `--fsid` option to `ceph-mon` [1] when [2] is run and propose a change for the demo pod.
I'll follow up on that part too

[1] https://docs.ceph.com/en/quincy/man/8/ceph-mon/
[2] https://github.com/ceph/ceph-container/blob/main/src/daemon/start_mon.sh